geode-redis/src/main/java/org/apache/geode/redis/internal/executor/server/InfoExecutor.java

```diff
@@ -28,6 +28,8 @@
 
 public class InfoExecutor extends AbstractExecutor {
 
+  private static final Long ONE_MEGABYTE = 1024 * 1024 * 1024L;
```

Comment:
I think 1 MB is `1024L x 1024L`, or 1048576 bytes... not sure if I'm missing something here (assuming the intention is to convert MB to bytes)?

Reply:
Good point. Until we can return real values from Geode it's not critical but we'll fix that ASAP!